client/signup/config/step-components.js

```diff
@@ -52,7 +52,7 @@ export default {
 	'domains-store': DomainsStepComponent,
 	'domain-only': DomainsStepComponent,
 	'domains-theme-preselected': DomainsStepComponent,
-	'import-from-url': ImportURLStepComponent,
+	'from-url': ImportURLStepComponent,
```

Comment:
`from-url` might seem ambiguous to someone outside of the project - would it be overkill to call it something like `import-get-url` or `import-url-input`?
Just thinking out loud so feel free to ignore